modules/migration-adding-cluster-to-cam.adoc

```diff
@@ -24,16 +25,24 @@ You can add a cluster to the {mtc-full} ({mtc-short}) web console.
 +
 [source,terminal]
 ----
-$ oc sa get-token migration-controller -n openshift-migration
+$ oc create token migration-controller -n openshift-migration
```

Comment:
@stillalearner - please can I ask you to confirm this change is correct? 

Thanks